main/Dockerfile

```diff
@@ -26,4 +26,4 @@ RUN crontab /etc/cron.d/syslog
 EXPOSE 602/tcp
 
 ENTRYPOINT ["/entrypoint.sh"]
-CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/conf.d/supervisord.conf"]
+CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/conf.d/supervisord.conf"]
```

Comment:
nitpick - IMO it's better to have a line terminator at the end of the file

Reply:
I can add it back in